### PR TITLE
textutil: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [textutil](https://ss64.com/osx/textutil.html) mac utility program.

### CODE REVIEW

File: pages/osx/textutil.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# textutil
+
+> Used to manipulate text files of various formats.
+
+- Display information about foo.rtf:
+
+`textutil -info foo.rtf`
```
Comment (mfrw) [inline pages/osx/textutil.md]:
```suggestion
 `textutil -info {{foo.rtf}}`
```

File: pages/osx/textutil.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert foo.rtf into foo.html:
+
+`textutil -convert html foo.rtf`
```
Comment (mfrw) [inline pages/osx/textutil.md]:
```suggestion
 `textutil -convert html {{foo.rtf}}`
```

File: pages/osx/textutil.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert rich text to normal text:
+
+`textutil foo.rtf -convert txt`
```
Comment (mfrw) [inline pages/osx/textutil.md]:
```suggestion
 `textutil {{foo.rtf}} -convert txt`
```

File: pages/osx/textutil.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert foo.txt into foo.rtf, using Times 10 for the font:
+
+`textutil -convert rtf -font Times -fontsize 10 foo.txt`
```
Comment (mfrw) [inline pages/osx/textutil.md]:
```suggestion
 `textutil -convert rtf -font Times -fontsize 10 {{foo.txt}}`
```

File: pages/osx/textutil.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Load all RTF files in the current directory, concatenates their contents, and writes the result out as index.html with the HTML title set to "Several Files":
+
+`textutil -cat html -title "Several Files" -output index.html *.rtf`
```
Comment (mfrw) [inline pages/osx/textutil.md]:
```suggestion
 `textutil -cat html -title {{"Several Files"}} -output {{index.html}} {{*.rtf}}`
```

**Comment by pxgamer** [comment]
@mfrw, was it intentional to add a space before each of the review suggestions? 🙂

File: pages/osx/textutil.md (hunk 1 above)
Comment (Aracki) [reply pages/osx/textutil.md]: OFC. my bad.

**Comment by mfrw** [comment]
> @mfrw, was it intentional to add a space before each of the review suggestions? 🙂

ohh nooo!!! something is wrong either with my machine or GitHub suggestions. It does not make any sense to add a space before each of the review suggestions and I really don't recall adding it :(

**Comment by pxgamer** [comment]
Weird. 🤔 Well, it looks like the changes have been applied without the prefixing spaces so all is good. 👍

**Pull request merged**